### PR TITLE
docs(migration): fixed broken link

### PR DESCRIPTION
**Pull request by sammarcus**
**Summary**

fixed a link that was broken

### CODE REVIEW

**Comment by sammarcus** [comment]
Hm, with [widgets.. as it is now it's not loading.

image

File: docgen/src/guides/migration.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ search.addWidget(instantsearch.widgets.searchbox({
 }));
 ```
 
-You can read more about these options [here](/widgets/searchBox.html).
+You can read more about these options [here](instantsearch.js/v2/widgets/searchBox.html).
```
Comment (bobylito) [inline docgen/src/guides/migration.md]: The migration page is already on `instantsearch.js/v2` therefore removing the `/` should be enough, no?

**Comment by bobylito** [comment]
@sammarcus do you have time to fix the comments?

File: docgen/src/guides/migration.md (hunk 1 above)
Comment (sammarcus) [reply docgen/src/guides/migration.md]: Sorry I hadn't seen your review earlier -- thank you @bobylito @Haroenv !

**Comment by bobylito** [comment]
Thanks a lot for your contribution @sammarcus 👍

**Pull request merged**